### DISCUSSION TITLE
Issue with load balancer since Version 2.8.2

### DISCUSSION THREAD

**Discussion by gigo1980** [mysticaltech/terraform-hcloud-kube-hetzner#1045] [Q&A]
Hi since upgrade my existing cluster to version 2.8.2 I have an issue that there was a new load balancer has been created.

Before we had "k3s", the new one is now "k3s-nginx". So far so good, but "k3s-nginx" does not map any node or create a service routing :/. The k3s still works, but the exported lb ipv4 ip is still the new one and this destroys my dns config.

Is this a known issue?

**Comment by Silvest89** [comment]
@gigo1980 
Delete the old loadbalancer in the hetzner console and then restart the hcloud-controller pod (this is what fixes the problem)
